classes/04class/exercises/c04-iac02/fabioacp/main.tfvars

```diff
@@ -0,0 +1,22 @@
+name = "devopsacademy"
+
+tags = {
+  Owner       = "FACP"
+  Environment = "DEV"
+  Project     = "C04-iac02"
+  BillingCode = "0800"
+}
+
+vpc_cidr = "10.0.0.0/16"
+
+azs                     = ["ap-southeast-2a", "ap-southeast-2b"]
+public_subnets          = ["10.0.1.0/24", "10.0.3.0/24"]
+public_subnet_names     = ["public-sb1", "public-sb2"]
+private_subnets         = ["10.0.3.0/24", "10.0.4.0/24"]
+private_subnet_names    = ["private-sb1", "private-sb2"]
```

Comment:
Tip: it was not requested in the exercise but you can get those values using the data resource aws_availability_zone, subnet_ids, etc:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/subnet_ids